cel/Cargo.toml

```diff
@@ -7,6 +7,7 @@ authors = ["Clark McCauley <me@clarkmccauley.com>", "Alex Snaps <alex@wcgw.dev>"
 edition = "2021"
 license = "MIT"
 categories = ["compilers"]
+rust-version = "1.87"
```

Comment:
🤔 I'll have a quick look at _what_ got us here... That's "quite recent" and if possible I'd rather have that not be the case

Reply:
Personally I'd say going back to 1.85 is good enough. Doing so gives the ability to use 2024 edition and have MSRV-aware cargo resolution.

It's the also version that comes with the current Debian stable (trixie).